apps/client/src/app/other-resources/other-resources-result/other-resources-result.component.ts

```diff
@@ -0,0 +1,212 @@
+import { Component, EventEmitter, OnInit } from '@angular/core';
+import { FhirService } from '../../shared/fhir.service';
+import { ActivatedRoute } from '@angular/router';
+import { DomainResource, OperationOutcome } from '../../../../../../libs/tof-lib/src/lib/stu3/fhir';
+import { getErrorString } from '../../../../../../libs/tof-lib/src/lib/helper';
+import { NgbModal, NgbTabChangeEvent } from '@ng-bootstrap/ng-bootstrap';
+import {saveAs} from 'file-saver';
+import { ChangeResourceIdModalComponent } from '../../modals/change-resource-id-modal/change-resource-id-modal.component';
+import { ConfigService } from '../../shared/config.service';
+import {Globals} from '../../../../../../libs/tof-lib/src/lib/globals';
+import { ValidatorResponse } from 'fhir/validator';
+
+class OpenedResource {
+  resource: DomainResource;
+  activeSub: 'json/xml'|'permissions' = 'json/xml';
+
+  constructor(resource) {
+    this.resource = resource;
+  }
+}
+
+@Component({
+  templateUrl: './other-resources-result.component.html',
+  styleUrls: ['./other-resources-result.component.css']
+})
+export class OtherResourcesResultComponent implements OnInit {
+  message: string;
+  data: OpenedResource;
+  Globals = Globals;
```

Comment:
Just an idea... not required for this. But, at some point it might be cool to create a BaseComponent (maybe we already have one but aren't using it everywhere) that has `Globals = Globals` defined in it, where all components can derive from it and automatically have access to Globals in the HTML template. Again, not required for this PR. But, just a future thought.